### DISCUSSION TITLE
How do I use a DLL in a C++ native module?

### DISCUSSION THREAD

**Discussion by meucontato** [microsoft/react-native-windows#12284] [Q&A] [answered]
I have some resources already developed and available in a dll and I didn't want to recreate everything new so my idea is to use the native C++ module to call this dll

**Comment by chrisglein** [comment, marked as answer]
Yes, this should be very possible. Native modules are just C++ code. Have you got as far as getting your basic native module (without linking in your DLL) yet? Once you're there, you should be able to use Visual Studio to link your DLL in just fine and call that code from your native module. Look for resources on this on docs.microsoft.com.
